serverless/messaging/api-cli/connect-aws-cli.mdx

```diff
@@ -42,21 +42,21 @@ pip3 install awscli-plugin-endpoint
 
 ## How to configure the AWS-CLI
 
-Now you have installed the AWS-CLI, you need to configure it for use with your Scaleway Messaging and Queuing SQS/SNS namespace.
+Now you have installed the AWS-CLI, you need to configure it for use with Scaleway Messaging and Queuing.
 
 1. Create a file named `~/.aws/config` by running the following command:
     ```
     aws configure set plugins.endpoint awscli_plugin_endpoint
```

Comment:
With a recent version of awscli (version 1.29.0 or 2.13.0), the use of a plugin is no longer necessary.

It's possible to directly define the endpoint_url:

```
[default]
region = fr-par
endpoint_url = https://sqs.mnq.fr-par.scaleway.com
```

It is easier. It's up to you to take this comment into consideration because you may want to be consistent with s3 documentation.